src/compass/utils/cloud_optimized_geotiff.py

```diff
@@ -0,0 +1,77 @@
+import os
+import shutil
+import tempfile
+import journal
+from osgeo import gdal, osr
+from compass.utils.validate_cloud_optimized_geotiff import main as validate_cog
+
+def save_as_cog(filename, scratch_dir = '.', flag_compress=True,
+                resamp_algorithm=None):
+    """Save (overwrite) a GeoTIFF file as a cloud-optimized GeoTIFF.
+       Parameters
+       ----------
+       filename: str
+              GeoTIFF to be saved as a cloud-optimized GeoTIFF
+       scratch_dir: str (optional)
+              Temporary Directory
+       flag_compress: bool (optional)
+              Flag to indicate whether images should be
+              compressed
+       resamp_algorithm: str (optional)
+              Resampling algorithm. Options: "AVERAGE",
+              "AVERAGE_MAGPHASE", "RMS", "BILINEAR",
+              "CUBIC", "CUBICSPLINE", "GAUSS", "LANCZOS",
+              "MODE", "NEAREST", or "NONE"
```

Comment:
I think I do not understand what is your suggestion here. Can you elaborate more on that?

Reply:
The suggestion (apologies for lack of comment for more context) was intended to add more details to the overview resampling behavior. I made a couple of mistakes:

* `MODE` -> `NONE` was mistake because I can not read.
* *Using `"None"` was confusing in line 446

I've fixed the suggestion and I hope my message is more clear.